src/components/QueryBuilder.vue

```diff
@@ -75,6 +75,19 @@ export default Vue.extend( {
 </script>
 
 <style scoped lang="scss">
+a {
```

Comment:
```suggestion
<!-- TODO replace with link component once available-->
a {
```

Reply:
This actually fails because this is not valid css :(